packages/api/models/rules.js

```diff
@@ -342,13 +342,15 @@ class Rule extends Manager {
         NextToken: token
       }).promise();
       token = subsResponse.NextToken;
-      /* eslint-disable no-loop-func */
-      await Promise.all(subsResponse.Subscriptions.forEach((sub) => {
-        if (sub.Endpoint === process.env.messageConsumer) {
-          subExists = true;
-          subscriptionArn = sub.SubscriptionArn;
-        }
-      }));
+      if (subsResponse.Subscriptions) {
```

Comment:
Was getting this error

```
{
    "error": {
        "name": "TypeError",
        "message": "Cannot read property 'Symbol(Symbol.iterator)' of undefined",
        "stack": [
            "TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined",
            "    at Function.all (<anonymous>)",
            "    at Rule.addSnsTrigger (/var/task/index.js:151904:23)",
            "    at <anonymous>",
            "    at process._tickDomainCallback (internal/process/next_tick.js:228:7)"
        ]
    },
    "level": "error",
    "message": "Cannot read property 'Symbol(Symbol.iterator)' of undefined",
    "sender": "unknown",
    "timestamp": "2018-11-15T20:07:11.101Z"
}
```

and this fixed it. But wasn't sure why this needed to be in a `Promise.all`, but I removed it and it worked.